django/db/backends/sqlite3/introspection.py

```diff
@@ -192,9 +192,9 @@ def get_primary_key_column(self, cursor, table_name):
         fields_sql = create_sql[create_sql.index('(') + 1:create_sql.rindex(')')]
         for field_desc in fields_sql.split(','):
             field_desc = field_desc.strip()
-            m = re.search('"(.*)".*PRIMARY KEY( AUTOINCREMENT)?', field_desc)
+            m = re.match('(?:(?:["`\[])(.*)(?:["`\]])|(\w+)).*PRIMARY KEY.*', field_desc)
```

Comment:
Hi all. This regex causes python to issue a deprecation warning.

```python
DeprecationWarning: invalid escape sequence \[
```

Reply:
I think the solution is to use a raw string: https://github.com/django/django/pull/9931.

Reply:
I can't remember - do the backslashes need to be removed? 

```python
# from this:
'(?:(?:["`\[])(.*)(?:["`\]])|(\w+)).*PRIMARY KEY.*'
#         ^             ^    (remove these)
# to this:
'(?:(?:["`[])(.*)(?:["`]])|(\w+)).*PRIMARY KEY.*'
```